cookbooks/aws-parallelcluster-platform/attributes/platform.rb

```diff
@@ -30,6 +30,7 @@
 default['cluster']['nvidia']['nvlsm']['enabled'] = true
 
 # DCV
+default['cluster']['dcv']['skip_install'] = false
```

Comment:
So far we have used the positive flavor: enabled v/s skipped.
So this would be: `['cluster']['dcv']['install_enabled'] = true`